src/main/kotlin/net/maizegenetics/phgv2/utils/LoggingUtils.kt

```diff
@@ -0,0 +1,43 @@
+package net.maizegenetics.phgv2.utils
+
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.core.config.builder.api.ConfigurationBuilderFactory
+
+private val myOriginalOutputStream = System.out
+private val myOriginalErrStream = System.err
+
```

Comment:
When do these get called?  Do each of the classes/files need to include a call to one of these functions?

Reply:
That's a good question. In TASSEL, I had the GUI and command line call it automatically, so that everything else didn't need to. But yeah, seems like each Clikt command will need to call the setup method. I wonder if we can put it in a central place so that units tests will automatically get it. I need to think about this more.

Reply:
Could it maybe go in PHG.kt? It looks like there is an init block which might be able to be used?

Reply:
@zrm22 Why didn't I think of that! Yes, done